### ISSUE TITLE
frame-ancestors not recognized with default-src 'none'

### ISSUE THREAD

**Issue by arigon** [mozilla/http-observatory-website#266]
Hello

when using `default-src 'none'` and no attribute for `frame-ancestors`, the tool will give a negative rating for `Clickjacking protection, using frame-ancestors`.

If I change it to `default-src 'none'; frame-ancestors 'none'` it will give a positive rating.

**Comment by rugk** [comment]
That is correct. [`frame-src](https://content-security-policy.com/frame-ancestors/) does **not** inherit from [`default-src`](https://content-security-policy.com/default-src/).

Maybe the message should be clarified though?

**Comment by gene1wood** [comment]
@rugk Can you propose a clarified message?

**Comment by rugk** [comment]
Hmm what is the current text?

**Comment by gene1wood** [comment]
I'm unsure but it should be discoverable in the code.
